### PR TITLE
xtest: fix cast issue in sdp_basic.c

### PR DESCRIPTION
**Pull request by etienne-lms**
Build issue found since the test is merged.
It's strange that I did not faced it before...

### CODE REVIEW

File: host/xtest/sdp_basic.c [hunk 1]
```diff
@@ -455,7 +455,7 @@ static int sdp_basic_test(enum test_target_ta ta,
 	fd = allocate_ion_buffer(sdp_size, ion_heap);
 	if (fd < 0) {
 		verbose("Failed to allocate SDP buffer (%lu bytes) in ION heap %d: %d\n",
-				sdp_size, ion_heap, fd);
+				(long unsigned)sdp_size, ion_heap, fd);
```
Comment (jforissier) [inline host/xtest/sdp_basic.c]: Use `%zu` instead?

**Comment by etienne-lms** [comment]
Pushed back the change, with "%zu" instead of casting size to "unsigned long". Looks better. Thanks Jerome.

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by etienne-lms** [comment]
rebased

**Pull request merged**